[2024-03-11 02:12:51,038] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-11T02:12:45.416761+00:00 [queued]>
[2024-03-11 02:12:51,053] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-11T02:12:45.416761+00:00 [queued]>
[2024-03-11 02:12:51,054] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-11 02:12:51,054] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-11 02:12:51,054] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-11 02:12:51,062] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-11T02:12:45.416761+00:00
[2024-03-11 02:12:51,067] {standard_task_runner.py:52} INFO - Started process 2239 to run task
[2024-03-11 02:12:51,070] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting_project492', 'merge_data', '2024-03-11T02:12:45.416761+00:00', '--job-id', '466', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpp9b1554x', '--error-file', '/tmp/tmpl4x1zajn']
[2024-03-11 02:12:51,072] {standard_task_runner.py:77} INFO - Job 466: Subtask merge_data
[2024-03-11 02:12:51,110] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting_project492.merge_data 2024-03-11T02:12:45.416761+00:00 [running]> on host fe457b6d1f83
[2024-03-11 02:12:51,158] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting_project492
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-11T02:12:45.416761+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-11T02:12:45.416761+00:00
[2024-03-11 02:12:51,200] {python.py:151} INFO - Done. Returned value was: None
[2024-03-11 02:12:51,208] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting_project492, task_id=merge_data, execution_date=20240311T021245, start_date=20240311T021251, end_date=20240311T021251
[2024-03-11 02:12:51,242] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-11 02:12:51,264] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
